binding/web/scripts/copy_ppn.js

```diff
@@ -1,23 +1,6 @@
 const fs = require("fs");
 const { join } = require("path");
 
-const ppnFiles = [
-  "alexa_wasm.ppn",
-  "americano_wasm.ppn",
-  "blueberry_wasm.ppn",
-  "bumblebee_wasm.ppn",
-  "computer_wasm.ppn",
-  "grapefruit_wasm.ppn",
-  "grasshopper_wasm.ppn",
-  "hey google_wasm.ppn",
-  "hey siri_wasm.ppn",
-  "jarvis_wasm.ppn",
-  "ok google_wasm.ppn",
-  "picovoice_wasm.ppn",
-  "porcupine_wasm.ppn",
-  "terminator_wasm.ppn",
-]
-
```

Comment:
when I run `copyppn` and `setup-test` it copies all the keywords from the wasm folder. We don't want to release most of them as built-ins and our tests don't really need most of them either. Can you change the script to only copy over this subset of the `_wasm.ppn` files?